script.service.janitor/addon.xml

```diff
@@ -39,15 +39,16 @@ Cet addon à la faculté de supprimer entièrement votre collection vidéo! Assu
         <disclaimer lang="nl_NL">Het is verstandig om videos pas permanent te verwijderen als je bekend bent met alle instellingen.</disclaimer>
         <disclaimer lang="fr_FR">Il est recommandé de ne pas supprimer physiquement de vidéos avant de bien maitriser les réglages.</disclaimer>
         <platform>all</platform>
+        <language>en nl fr</language>
         <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```

Comment:
would be nice if you could use the spdx identifier for the license instead (GPL-3.0-only or GPL-3.0-or-later depending on your license